week05/src/TodoList.js

```diff
@@ -0,0 +1,21 @@
+import TodoItem from "./TodoItem";
+
+function TodoList({ todoList, setTodoList }) {
+    return (
+        <div className="todo-list">
+            <ul>
+                {todoList.map((todoItem) => {
+                    return (
+                        <TodoItem
+                            todoThis={todoItem}
+                            todoList={todoList}
+                            setTodoList={setTodoList}
+                        />
+                    );
+                })}
```

Comment:
P1: map 함수를 사용할 때에는 그 내부의 child에 unique한 id(key)를 반드시 부여해줘야 합니다! 에러는 안나지만, 아마 개발자모드의 콘솔에서 확인하면 warning이 뜰 거예요. 다음과 같이 추가해주세요~
```suggestion
{todoList.map((todoItem) => {
          return (
            <TodoItem
              key={todoItem.id}
              todoThis={todoItem}
              todoList={todoList}
              setTodoList={setTodoList}
            />
          );
        })}
```